tests/cli_test.go

```diff
@@ -0,0 +1,206 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/sigstore/timestamp-authority/pkg/client"
+)
+
+var (
+	restapiURL string
+)
+
+const (
+	cli = "../timestamp-cli"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Looking into restructuring these tests as subtests to see if we can get around `TestMain`.